pom.xml

```diff
@@ -260,6 +263,21 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
Can we use `dependencyManagement` to overwrite transitive dependency versions? Then we can also remove the manual exclusions we did as part of the vulnerability fixes

Reply:
In this case, the dependences versions were missing from the JAR due to previous exclusions.

Reply:
Right, in that case `dependencyManagement` can allow us to overwrite those transitive dependencies without having to manually exclude them, i.e. we can remove the exclusions added in https://github.com/data-integrations/amazon-s3-plugins/pull/141.

Reply:
See https://github.com/cdapio/hydrator-plugins/pull/1747 for an example.